exporters/etw/src/etw_tracer_exporter.cc

```diff
@@ -0,0 +1,7 @@
+#ifdef _WIN32
+
+#  define HAVE_NO_TLD
+
+#  include "opentelemetry/exporters/etw/etw_tracer_exporter.h"
```

Comment:
Is it necessary to have this in the compilation unit? Or could we go with a header-only exporter here?

Otherwise, could we just have class definitions in the headers and move the implementation here?

Reply:
This was originally made header-only. I think the reason why it got included in source to satisfy the unit testing requirements. I don't have an answer. @mishal23 may clarify why he needed this.

Reply:
yeah, not needed. I kept it with the aim to move the implementations from the header class to here, but not needed right now, since header only is expected.

Reply:
Compiling header-only code in compilation units should be avoided.